step/step.go

```diff
@@ -0,0 +1,553 @@
+package step
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/go-utils/progress"
+	"github.com/bitrise-io/go-utils/retry"
+	"github.com/bitrise-steplib/steps-xcode-test/cache"
+	"github.com/bitrise-steplib/steps-xcode-test/output"
+	"github.com/bitrise-steplib/steps-xcode-test/simulator"
+	"github.com/bitrise-steplib/steps-xcode-test/xcodebuild"
+	"github.com/bitrise-steplib/steps-xcode-test/xcpretty"
+)
+
+const (
```

Comment:
Should we merge `const` declarations here (separated by empty lines for sections)? Otherwise, I'd suggest removing parentheses where it has only one declaration.